hash/amigaocs_flop.xml

```diff
@@ -28534,7 +28534,7 @@
 
 	<software name="millen" supported="no">
 		<!-- SPS (CAPS) release 2027 -->
-		<description>Millenium - Return to Earth (USA)</description>
+		<description>Millennium - Return to Earth (USA)</description>
```

Comment:
The game is called "Millenium - Return to Earth". Yes, it's spelled incorrectly. See http://hol.abime.net/5159.